docs/lo/docs/index.md

```diff
@@ -457,13 +458,13 @@ Used by Starlette:
 * <a href="https://pyyaml.org/wiki/PyYAMLDocumentation" target="_blank"><code>pyyaml</code></a> - Required for Starlette's `SchemaGenerator` support (you probably don't need it with FastAPI).
 * <a href="https://github.com/esnme/ultrajson" target="_blank"><code>ujson</code></a> - Required if you want to use `UJSONResponse`.
 
-Used by FastAPI / Starlette:
+ໃຊ້ໂດຍ FastAPI / Starlette:
 
 * <a href="https://www.uvicorn.org" target="_blank"><code>uvicorn</code></a> - for the server that loads and serves your application.
 * <a href="https://github.com/ijl/orjson" target="_blank"><code>orjson</code></a> - Required if you want to use `ORJSONResponse`.
```

Comment:
Thanks for the effort! 👏 

It seems there are still sections not translated yet, right?

Reply:
Yes, that's right. I will translate other section later on